File: GTTS/INL.MessagingService/Models/Notification_Item.cs
using System;
using System.Collections.Generic;

namespace INL.MessagingService.Models
{
    public class Notification_Item : INotification_Item
    {
        public long NotificationID { get; set; }
        public string NotificationSubject { get; set; }
        public string NotificationMessage { get; set; }
        public string EmailMessage { get; set; }
        public bool Unread { get; set; }
        public int NotificationContextTypeID { get; set; }
        public string NotificationContextType { get; set; }
        public long ContextID { get; set; }
        public DateTime ModifiedDate { get; set; }
        public int NotificationMessageID { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public string ModifiedByAppUser { get; set; }
        public string MessageTemplateName { get; set; }
        public bool IncludeContextLink { get; set; }
        public int AppUserID { get; set; }
        public DateTime? ViewedDate { get; set; }
        public DateTime? EmailSentDate { get; set; }
        public DateTime NotificationRecipientModifiedDate { get; set; }

        public List<NotificationRecipient_Item> Recipients { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.MessagingService.Models/notification_item.ts


import { NotificationRecipient_Item } from './notification-recipient_item';
import { INotification_Item } from './inotification_item';

export class Notification_Item implements INotification_Item {
  
	public NotificationID: number = 0;
	public NotificationSubject: string = "";
	public NotificationMessage: string = "";
	public EmailMessage: string = "";
	public Unread: boolean = false;
	public NotificationContextTypeID: number = 0;
	public NotificationContextType: string = "";
	public ContextID: number = 0;
	public ModifiedDate: Date = new Date(0);
	public NotificationMessageID: number = 0;
	public ModifiedByAppUserID: number = 0;
	public ModifiedByAppUser: string = "";
	public MessageTemplateName: string = "";
	public IncludeContextLink: boolean = false;
	public AppUserID: number = 0;
	public ViewedDate?: Date;
	public EmailSentDate?: Date;
	public NotificationRecipientModifiedDate: Date = new Date(0);
	public Recipients?: NotificationRecipient_Item[];
  
}


